cli/src/commands/list.ts

```diff
@@ -12,28 +13,51 @@ export default class List extends Command {
 
   public static strict = true;
 
+  public static flags = {
+    ...Command.flags,
+    format: flags.string({
+      char: 'f',
+      description: 'Format output',
+      default: 'table',
+      options: ['table', 'json'],
+    }),
+  };
+
   public async run() {
-    this.parse(List);
+    const { format } = this.parse(List).flags;
     await this.authenticate();
     const apps = await this.lycanClient.listApps();
-    if (apps.length === 0) {
-      this.log('You do not have any apps yet.');
-      return;
-    }
-    this.log(columnify(apps.map(({ name, updatedAt, environments }) => ({
-      name, updatedAt: updatedAt.toISOString(), URL: `https://${environments[0].domains[0].name}`,
-    })), {
-      columns: ['name', 'updatedAt', 'URL'],
-      config: {
-        name: {
-          headingTransform: () => 'APPLICATION',
-          minWidth: 25,
-        },
-        updatedAt: {
-          headingTransform: () => 'LAST UPDATED',
-          minWidth: 25,
-        },
-      },
+    const mappedApps = apps.map(({ name, updatedAt, environments }) => ({
+      name,
+      updatedAt: updatedAt.toISOString(),
+      URL: `https://${environments[0].domains[0].name}`,
```

Comment:
This is an unfortunate itnerface: if ever we change name of the app's page (a *remote* change), old CLIs will give the wrong URL.  Add another field (in another PR)?

Reply:
What do you mean by name of the app's page?